CONTRIBUTING.md

```diff
@@ -131,24 +133,50 @@ when creating documentation. Refer to our [Component Guide](COMPONENT_GUIDE.md)
 4. If you are willing to assist in the process of creating a new page, then follow the instructions above to `Fork` or `Clone` the repo and make your edits.
 5. Follow the instructions above to submit a `PR` for your change.
 
-### Adding a "what's new" post
+### Adding a what's new post
 
-We use "What's new" posts, or NR1 announcements, to let people know about new features and updates in [New Relic One](https://one.newrelic.com/launcher/jerome.plg-whats-new-launcher?pane=eyJuZXJkbGV0SWQiOiJqZXJvbWUucGxnLXdoYXRzLW5ldyJ9). You can also find What's new on our [Docs site](https://docs.newrelic.com/whats-new). To add a new post:
+New Relic use "what's new" posts, or NR1 announcements, to let our customers know about new features and updates in the [New Relic One](https://one.newrelic.com/launcher/jerome.plg-whats-new-launcher?pane=eyJuZXJkbGV0SWQiOiJqZXJvbWUucGxnLXdoYXRzLW5ldyJ9) product. You can also find What's new on our [Docs site](https://docs.newrelic.com/whats-new).
 
-1. If it doesn't exist, create the folder for the current year and month in the [`src/content/whats-new/`](https://github.com/newrelic/docs-website/tree/develop/src/content/whats-new) directory (i.e. `src/content/whats-new/2020/12` for December 2020).
-2. Create a markdown file (`.md`) in the right year and month directory (**note:** this is _not_ a MDX file).
-3. Add this to the top of your markdown file. This is the frontmatter for What's new posts.
+To add a new post:
+
+1. If the post doesn't exist, create the folder for the current year and month in the [`src/content/whats-new/`](https://github.com/newrelic/docs-website/tree/develop/src/content/whats-new) directory (e.g. `src/content/whats-new/2020/12` for December 2020). **note:** the year and month definition will determine where to place the post in the navigation on the [Docs site](https://docs.newrelic.com/whats-new).
+2. Create a markdown file (`.md`) in the correct year and month directory (**note:** this is _not_ a MDX file).
+3. Add the correct frontmatter to the top of your markdown file between a set of `---` which specifies a frontmatter block.
```

Comment:
I'd rewrite this slightly:

"Add the correct frontmatter to the top of your markdown file between a set of `---`. This marks it as a frontmatter content block."